Sources/Apodini/Semantic Model Builder/gRPC/GRPCService/GRPCService.swift

```diff
@@ -23,14 +29,77 @@ class GRPCService {
         self.serviceName = name
         self.app = app.vapor.app
     }
+
+    internal func checkContentType(request: Vapor.Request) -> Bool {
+        // GRPC theoretically would also allow for other
+        // types of payload formats, e.g. JSON.
+        // We do only support proto payloads at the moment
+        if request.content.contentType == Self.grpc ||
+            request.content.contentType == Self.grpcproto {
+            return true
+        }
+        return false
+    }
```

Comment:
```suggestion
    internal func contentTypeIsSupported(request: Vapor.Request) -> Bool {
        // GRPC theoretically would also allow for other
        // types of payload formats, e.g. JSON.
        // We only support proto payloads at the moment
        request.content.contentType == Self.grpc || request.content.contentType == Self.grpcproto
    }
```

This should read better in an if-statement, no?